android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -50,6 +54,8 @@
   private BraveShieldsContentSettings mBraveShieldsContentSettings;
   private BraveShieldsContentSettingsObserver mBraveShieldsContentSettingsObserver;
 
+  static final String PREF_HIDE_BRAVE_REWARDS_ICON = "hide_brave_rewards_icon";
```

Comment:
How about using `AppearancePreferences.PREF_HIDE_BRAVE_REWARDS_ICON` here instead of dupe?

Reply:
https://github.com/brave/brave-core/pull/3665/commits/7b3065634dbf1ed12ae3a8f6cdac7433aa290baa